lisa/schema.py

```diff
@@ -724,6 +783,7 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         default=search_space.IntRange(min=512),
         metadata=field_metadata(decoder=search_space.decode_count_space),
     )
+    acc: Optional[ACCOptionSettings] = None
```

Comment:
it should go with other features, not in the NodeSpace directly.

Reply:
This comment is not applicable since we removed settings